tests/test_stumpff.py

```diff
@@ -18,14 +19,14 @@ def test_stumpff_functions_above_zero():
     expected_c2 = (1 - cos(psi**0.5)) / psi
     expected_c3 = (psi**0.5 - sin(psi**0.5)) / psi**1.5
 
-    assert_equal(c2(psi), expected_c2)
-    assert_equal(c3(psi), expected_c3)
+    assert c2(psi) == pytest.approx(expected_c2, rel=1e-10)
+    assert c3(psi) == pytest.approx(expected_c3, rel=1e-10)
```

Comment:
I just realized, shouldn't we use here `numpy.testing.assert_allclose`? This function is already imported at the very top of this test module.

Reply:
Sure, that's more a matter of personal preference, but as `poliastro` is already using the `numpy.testing` framework I'll swap over to that over `pytest`'s. I'll do a quick forepush on this in a few minutes. :+1:

Reply:
Thanks a lot, approving and merging this 🚀